pygamelib/engine.py

```diff
@@ -91,7 +92,7 @@ def __init__(self, **kwargs):
         # The overlapped matrix is used as an invisible layer were overlapped
         # restorable items are parked momentarily (until the cell they were on
         # is free again).
-        self._overlapped_matrix = []
+        self._overlapped_matrix = np.array([])
```

Comment:
We should probably init self._matrix with an empty array too. I never paid attention to that until today...

Reply:
Done

Reply:
Making this better on the new version